libs/ui-lib-tests/cypress/fixtures/create-mn/requests.ts

```diff
@@ -24,11 +24,11 @@ const UMNetworkingRequest = {
   network_type: 'OVNKubernetes',
 };
 
-const NetworkingRequest = {
-  api_vips: [{ ip: '192.168.122.10', cluster_id: fakeClusterId }],
-  api_vip: '192.168.122.10',
-  ingress_vips: [{ ip: '192.168.122.110', cluster_id: fakeClusterId }],
-  ingress_vip: '192.168.122.110',
+const getNetworkingRequest = (apiVip: string, ingressVip: string) => ({
```

Comment:
I had changed it because since I signalled `READY_TO_INSTALL`, those Vips were equal to the default ones, so the test had to verify that the entered Vips were submitted.
Since I have changed the tests to use two describe blocks, I've also reverted this change.